pkg/cmd/codespace/logs.go

```diff
@@ -42,6 +44,19 @@ func (a *App) Logs(ctx context.Context, selector *CodespaceSelector, follow bool
 		return err
 	}
 
+	cmdType := "cat"
+	if follow {
+		cmdType = "tail -f"
+	}
+
+	args := []string{fmt.Sprintf("%s /workspaces/.codespaces/.persistedshare/creation.log", cmdType)}
+
+	keys := sshkey.NewConfig(ssh.Context{})
+	keyPair, args, err := keys.FindKey(ctx, args, "")
```

Comment:
This is going to always select the automatic key, since there is never a `-i` flag added. That usually works, but we've gotten complaints in the past that users who got flagged for having the automatic key on their work machines.

Should we add an optional parameter for people to workaround that? Or support `gh cs logs -- <ssh-args>` like we do with `ssh` and `cp`?

Alternatively, I think @dmgardiner25 has added a gRPC call to get the content of a file that we could use here instead of `ssh cat`. That wouldn't support `--follow`, but anyways the command is totally busted so I think it might be OK to just remove that option?

Reply:
Makes sense! Let me work on that feedback and I'll let you know.

Reply:
I was thinking about this more overnight: something to consider actually is that this is only going to export the devcontainer creation log - which is not going to be super helpful for your goal of exporting all the codespace logs for debugging. We should probably switch this to using the full `ExportLogs` grpc call rather than just retrieving this one file

Reply:
I had not seen this comment, but after the refactor I've done it should be relatively easy to add a new option to the logs command that does just this.